.circleci/config.yml

```diff
@@ -11,7 +11,7 @@ jobs:
     - run:
         name: Build native
         command: |
-          docker-compose run Datadog.Trace.ClrProfiler.Native
+          docker-compose run Profiler
```

Comment:
We don't use CircleCI for this project. Feel free to delete the entire `.circleci` directory.